pydicom/filewriter.py

```diff
@@ -736,9 +736,16 @@ def dcmwrite(filename, dataset, write_like_original=True):
     the DICOM File Format).
 
     If `write_like_original` is ``False``, `dataset` will be stored in the
-    :dcm:`DICOM File Format <part10/chapter_7.html>`. The
-    byte stream of the `dataset` will be placed into the file after the
-    DICOM *File Meta Information*.
+    :dcm:`DICOM File Format <part10/chapter_7.html>`.  To do
+    so requires that the ``Dataset.file_meta`` attribute
+    exists and contains a :class:`Dataset` with the required (Type 1) *File
+    Meta Information Group* elements. The byte stream of the `dataset` will be
+    placed into the file after the DICOM *File Meta Information*.
+
+    If `write_like_original` is ``True`` then the :class:`Dataset` will be
+    written as is (after minimal validation checking) and may or may not
+    contain all or parts of the *File Meta Information* (and hence may or
+    may not be conformant with the DICOM File Format).
 
```

Comment:
Maybe you can add a description of the exceptions raised (e.g. `AttributeError`, `KeyError`).

Reply:
Sounds good, will do.

Reply:
KeyError was trapped and ignored.  Did ValueError and AttributeError.

Reply:
Yes, correct - I didn't notice. Thanks!